### ISSUE TITLE
does not load in LispWorks 7.1.1

### ISSUE THREAD

**Issue by lispm** [metawilm/cl-python#23]
```
CL-USER 1 > (ql:quickload "clpython")
...
**++++ Error in (DEFCLASS CLPYTHON:PY-TYPE): 
  #<STANDARD-CLASS DICT-MIXIN 40200E136B> is an invalid superclass of #<PY-META-TYPE PY-TYPE 402013771B>.
```

**Comment by drmeister** [comment]
The same sort of error happens in Clasp and in ECL.

We've traced it down to how these lisps handle DEFCLASS at compile time.
Clasp and ECL share CLOS code - and DEFCLASS creates classes at compile-time.
The problem is that the specialized versions of validate-superclasses that clpython defines to handle these classes aren't available at compile time.  This is more the fault of the implementation than clpython - we are looking into it.

**Comment by drmeister** [comment]
I was too vague - the exact error happens with Clasp and ECL.

From ECL:

;;; Error:
;;;   in file metaclass.lisp, position 1141
;;;   at (DEFCLASS PY-TYPE ...)
;;;   * The form (PROGN (PROGN (EXT:OPTIONAL-ANNOTATION 'PY-TYPE 'EXT:LOCATION '(DEFCLASS PY-TYPE) '(#P"/Users/meister/quicklisp/dists/quicklisp/software/cl-python-20181210-git/runtime/metaclass.lisp" . 1141)) NIL (CLOS:LOAD-DEFCLASS 'PY-TYPE '(DICT-MIXIN STANDARD-CLASS) NIL (LIST :METACLASS 'PY-META-TYPE)))) was not evaluated successfully.
;;; Error detected:
;;; Class #<The STANDARD-CLASS CLPYTHON::DICT-MIXIN> is not a valid superclass for #<The CLPYTHON::PY-META-TYPE CLPYTHON:PY-TYPE>
Condition of type: COMPILE-FILE-ERROR
COMPILE-FILE-ERROR while compiling #<cl-source-file "clpython/runtime" "runtime" "metaclass">

Available restarts:

1. (RETRY) Retry compiling #<cl-source-file "clpython/runtime" "runtime" "metaclass">.
2. (ACCEPT) Continue, treating compiling #<cl-source-file "clpython/runtime" "runtime" "metaclass"> as having been successful.
3. (RETRY) Retry ASDF operation.
4. (CLEAR-CONFIGURATION-AND-RETRY) Retry ASDF operation after resetting the configuration.
5. (ABORT) Give up on "clpython"
6. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at SI:BYTECODES. [Evaluation of: (QUICKLISP-CLIENT:QUICKLOAD :CLPYTHON)] In: #<process TOP-LEVEL 0x106706f80>.
>>

**Comment by metawilm** [comment]
Thanks for the report. This should be fixed by commits cd01f896 and 8e8053a3.
